lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -314,7 +314,9 @@ protected function transformCampaign($data) {
 
         $output['staff_pick'] = $data->staff_pick;
 
-        $output['facts'] = $data->facts;
+        $output['facts']['problem'] = $data->facts['problem'] ? $data->facts['problem']['fact'] : NULL;
+        $output['facts']['solution'] = $data->facts['solution'] ? $data->facts['solution']['fact'] : NULL;
+        $output['facts']['sources'] = $data->facts['sources'] ?: NULL;
```

Comment:
I think shorthand makes this less readable. Drupal standards recommend always using `{`'s: https://www.drupal.org/coding-standards#controlstruct